api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -407,8 +406,7 @@ void shouldTestClaimRemainingPendingExpiryIfLessThan16Weeks(List<LocalDate> prev
                 Arguments.of(SINGLE_THREE_YEAR_OLD, SINGLE_THREE_YEAR_OLD, DUE_DATE_IN_4_MONTHS),
                 Arguments.of(NO_CHILDREN, SINGLE_THREE_YEAR_OLD, DUE_DATE_IN_4_MONTHS),
                 Arguments.of(SINGLE_THREE_YEAR_OLD, NO_CHILDREN, DUE_DATE_IN_4_MONTHS),
-                Arguments.of(NO_CHILDREN, NO_CHILDREN, DUE_DATE_IN_4_MONTHS),
-                Arguments.of(SINGLE_THREE_YEAR_OLD, NO_CHILDREN, NOT_PREGNANT)
```

Comment:
This integration test scenario was invalid for HTBHF-1757 - if the claimant has no children in the current cycle and is not pregnant then that scenario is the same as that for this story, hence they should be set to EXPIRED and sent an email to say they are no longer on the service. Have removed this scenario from that parameterised test.

Reply:
Wasn't there a difference between sudden loss of a child (i.e. last cycle there was a child who would still be under 4) and the last child turning 4?

Reply:
At the moment, this scenario results in claim being set to EXPIRED and the claimant being sent an email to say that they are no longer eligible because they have no children and are not pregnant, so yes, it is different to their child turning 4, but this scenario doesn't fit in with these tests and hence why I removed it, does that make sense?